plugins/embedbase/plugin.js

```diff
@@ -424,7 +424,7 @@
 						request.task.done();
 					}
 
-					this._setContent( request.url, evtData.html );
+					this._setContent( request.url, CKEDITOR.tools.htmlSafeByReview(evtData.html, 'We assume that evtData is safe and no code can be injected here.') );
```

Comment:
Legacy since we're not sure the eventData is safe.

Reply:
Legacy converted